test/common/FileEncoding/FileEncoding.Tests.ps1

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
We should actually run this test somewhere

Reply:
Perhaps, add another job to here
https://github.com/PowerShell/PowerShell/blob/master/.vsts-ci/misc-analysis.yml

**Or** add another workflow:

[workflows](https://github.com/PowerShell/PowerShell/tree/master/.github/workflows)

Whichever you are more comfortable with